src/aks-preview/azcli_aks_live_test/vsts-azcli-aks-live-test.yaml

```diff
@@ -5,15 +5,15 @@ trigger: none
 pr:
   branches:
     include:
-    - master
+    - main
   paths:
     include:
     - src/aks-preview/
 
 jobs:
 - job: LiveTest
   pool:
-    vmImage: 'ubuntu-16.04'
+    vmImage: 'ubuntu-latest'
```

Comment:
I don't think this is a good idea. we should always stick to one version.

Reply:
Fixed in commit [4f71818](https://github.com/Azure/azure-cli-extensions/pull/3532/commits/4f71818df70f3a4e3a1aaad4cac70b541f700738).